### PR TITLE
Add 3.7.1 Release Notes

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
* [ ] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (planned Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [ ] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8; orcharhino 6.3 on EL7/8)
* We do not accept PRs for Foreman older than 3.1.

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
Let me know if I messed up the contributors section. It looked like it hadn't been updated since 3.7.0-rc1. I ran it from 3.6.0 to 3.7.1. Is that correct?

File: guides/doc-Release_Notes/topics/foreman-contributors.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+Adam Hosek, Adam Ruzicka, Alex Fisher, Alexey Masolov, Andrew Teixeira, Antonin Dvorak, Archana Kumari, Areyus, Bastian Schmidt, Bernhard Suttner, Damon Clinkscales, Daniel Alley, Dirk Götz, Eric Cook, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Girija Soni, Gordon Bleux, Griffin Sullivan, Ian Ballou, Jan Klare, Jerome Charaoui, Joshua Hoblitt, Leos Stejskal, Lior Keren, Lucy Fu, Marek Hulán, Maria Agaphontzev, Markus Bucher, Maximilian Kolb, Nagoor Shaik, Nofar Alfassi, Oleh Fedorenko, Pat Riehecky, Peter Bittner, Romain Tartière, Ron Lavi, Samir Jha, Sebastian Bublitz, Tim Meusel, Trey Dockendorf, William Clark, archanaserver, chr1s692, girijaasoni, jc, pkoprda, vijaysawant
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman-contributors.adoc:3]: There are a few that we don't properly map here. `girijaasoni` is `Girija Soni`. There are a few others we should expand. At least `archanaserver` and `pkoprda` are Red Hat colleagues. IIRC `vijaysawant` was too.

Comment (Griffin-Sullivan) [reply guides/doc-Release_Notes/topics/foreman-contributors.adoc:3]: ok let me try running it again but fix the mapping

File: guides/doc-Release_Notes/topics/foreman-3.7.1.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+* pass:[Wrong initrd extension for preseed_default_ipxe] - https://projects.theforeman.org/issues/36847[#36847]
+
+=== Host groups
+
+* pass:[500 error when loading Hostgroups page for Ansible Roles Manager user] - https://projects.theforeman.org/issues/36703[#36703]
+
+=== Host registration
+
+* pass:[global registration should not create hosts as "managed" or "to be built"] - https://projects.theforeman.org/issues/36393[#36393]
```
Comment (mjivraja) [inline guides/doc-Release_Notes/topics/foreman-3.7.1.adoc:15]:
```suggestion
* pass:[Global registration should not create hosts as "managed" or "to be built"] - https://projects.theforeman.org/issues/36393[#36393]
```

File: guides/doc-Release_Notes/topics/foreman-contributors.adoc [hunk 3]
```diff
@@ -1,5 +1,4 @@
...
-
+Adam Hosek, Adam Ruzicka, Alex Fisher, Alexey Masolov, Andrew Teixeira, Antonin Dvorak, Archana Kumari, Areyus, Bastian Schmidt, Bernhard Suttner, Damon Clinkscales, Daniel Alley, Dirk Götz, Eric Cook, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Girija Soni, Gordon Bleux, Griffin Sullivan, Ian Ballou, Jan Klare, Jerome Charaoui, Joshua Hoblitt, Leos Stejskal, Lior Keren, Lucy Fu, Marek Hulán, Maria Agaphontzev, Markus Bucher, Maximilian Kolb, Nagoor Shaik, Nofar Alfassi, Oleh Fedorenko, Pat Riehecky, Peter Bittner, Peter Koprda, Romain Tartière, Ron Lavi, Samir Jha, Sebastian Bublitz, Tim Meusel, Trey Dockendorf, Vijaykumar Sawant, William Clark, chr1s692, jc
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/foreman-contributors.adoc:3]: Somehow I no longer see Anthony Somerset here.

Recently @wbclark pointed out that you can put everyone on their own line and in the rendered document it ends up as a single sentence. I think the best approach is to replace it first, commit that and then replace the whole set. The line by line comparison should make it very easy to see.

So use `sed -i 's/, /,\n/g' guides/doc-Release_Notes/topics/foreman-contributors.adoc`.

I've also opened https://github.com/theforeman/theforeman.org/pull/2112 to automate the replacement of names.

Comment (Griffin-Sullivan) [reply guides/doc-Release_Notes/topics/foreman-contributors.adoc]: ok I placed them on their own lines. Was I supposed to add Anthony back?

**Comment by Griffin-Sullivan** [comment]
I just rebased this. Not sure what the next step should be @ekohl

**Review by pcreech — APPROVED** [review]

File: guides/doc-Release_Notes/topics/foreman-contributors.adoc (hunk 3 above)
Comment (pcreech) [reply guides/doc-Release_Notes/topics/foreman-contributors.adoc]: I don't have enough context to explicitly know of Anthony was, but I would assume that anyone present as a contributor in 3.7.0 should also be a contributor in 3.7.1

**Review by ekohl — APPROVED** [review]

**Pull request merged**